### PR TITLE
Improve reporting

### LINKED ISSUE #193: Install failure and timeouts should go in separate failures.md
Rather than interleaving with `problems.md`

### PR DESCRIPTION
**Pull request by hadley**
Including separate report for failures. Fixes #193

### CODE REVIEW

**Comment by hadley** [comment]
Fixed now

File: R/report.R [hunk 1]
```diff
@@ -59,13 +64,37 @@ revdep_report_section <- function(title, rows, file) {
...
+
+#' `revdep_report_failures()` generates a report about packages that failed
+#' to check (i.e. couldn't install, or timed out).
+#'
+#' @export
+#' @rdname revdep_report_summary
+
+revdep_report_failures <- function(pkg = ".", file = "", results = NULL) {
+  problem <- function(x) {
+    !x$status %in% c("+", "-")
```
Comment (gaborcsardi) [inline R/report.R]: So we want to show everything here? Even if (say) the new check ran without errors?

Comment (hadley) [reply R/report.R]: Isn't that `+`? This should include all non-check failures. It did what I expected in my tests, but I don't know what values status can take.

Comment (gaborcsardi) [reply R/report.R]: `t+` means that the new check was good, but the old timed out
`i+` means that the new check was good, but the old failed to install
`t-` means that at least one timed out, and new check was bad
`i-` means that at least one failed to install and new check was bad

Comment (hadley) [reply R/report.R]: Ok, I think I'm primarily interested `t-` and `i-`. What if both checks timed out

Comment (gaborcsardi) [reply R/report.R]: `t-` is both timed out.

Comment (gaborcsardi) [reply R/report.R]: basically a `-` means that we could not check with the new version.

**Comment by gaborcsardi** [comment]
Thanks!

**Pull request merged**